staff/joel-molero/mailchimp/styles.sass

```diff
@@ -0,0 +1,162 @@
+header
+    background-color: rgb(74,189,210)
+
+.li
+    display: inline-block
+    background-color: #DEF3F7
+    padding: 12px 15px
+    margin: -2px
+    border-radius: 10px 0px 0px 10px
+    font-family: 'Work Sans', sans-serif
+
+.signupfree
+    padding: 12px 15px
+    background-color: #2E3233
+    border-radius: 10px
+    color: white
+    border: none
+   
+.li2
+    display: inline-block    
+    background-color: #DEF3F7
+    padding: 12px 15px
+    margin: -2px
+    font-family: 'Work Sans', sans-serif
+
+.li3
+    display: inline-block    
+    background-color: #DEF3F7
+    padding: 12px 15px
+    margin: -2px
+    border-radius: 0px 10px 10px 0px
+
+.link
+    color: black
+    text-decoration: none
+
+.link2
+    color: #535859
+    text-decoration: none
+    font-family: 'Work Sans', sans-serif
+
+.footerdiv
+    display: flex
+    flex-direction: column
+    align-items: center
+
+
+.search
+    display: inline
+
+.topbar
+    display: flex
+    align-items: center
+    align-content: space-around
+
+main
+    position: relative
+    background-image: url("./images/img/noisepattern.png")
+    height: 600px
+
+
+.title-img
+    display: flex
+
+.menu
+    display: flex
+    justify-content: space-around
+
+footer
+    display: flex
+    flex-direction: column
+    background-color: black
+    justify-content: space-around
+
+.footernavs
+    display: flex
+    justify-content: space-around
+    flex-direction: row
+    height: 300px
+
+.footerp
+    color: white
+    text-align: center
+    padding-bottom: 30px
+    font-family: 'Work Sans', sans-serif
```

Comment:
I noticed you mentioned the same font-family in multiple places. Maybe you can declare it in a more general selector? or store it as a variable?

Reply:
I like it this way to keep it more visual for me and easier to manipulate.